File: src/test/java/com/coors/StudentTest.java
package com.coors;

import com.coors.student.Student;
import org.junit.Test;
import org.junit.jupiter.api.Assertions;

public class StudentTest {

    @Test
    public void getAverageTest() {
        Student student = new Student("coors", 80 ,96);

        Assertions.assertEquals(88,student.getAverage());
    }

    @Test
    public void getHighestTest() {
        Student student = new Student("coors", 80 ,96);

        Assertions.assertEquals(96,student.getHighest());
    }
}

File: src/test/kotlin/com/coors/StudentTest.kt
package com.coors.kotlin

import com.coors.student.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {

    @Test
    fun highestTest() {
        val student = Student("coors" , 80 , 96)
        Assertions.assertEquals(96,student.highest)
    }

    @Test
    fun averageTest() {
        val student = Student("coors" , 80 , 96)
        Assertions.assertEquals(88,student.average)
    }
}
